velox/connectors/hive/storage_adapters/abfs/tests/AzuriteServer.cpp

```diff
@@ -15,15 +15,33 @@
  */
 
 #include "velox/connectors/hive/storage_adapters/abfs/tests/AzuriteServer.h"
+#include "velox/connectors/hive/storage_adapters/abfs/AbfsConfig.h"
 
-namespace facebook::velox::filesystems::test {
-const std::string AzuriteServer::connectionStr() const {
+namespace facebook::velox::filesystems {
+
+std::string AzuriteServer::URI() const {
   return fmt::format(
-      "DefaultEndpointsProtocol=http;AccountName={};AccountKey={};BlobEndpoint=http://127.0.0.1:{}/{};",
-      AzuriteAccountName,
-      AzuriteAccountKey,
-      port_,
-      AzuriteAccountName);
+      "abfs://{}@{}.dfs.core.windows.net/", container_, account_);
+}
+
+std::string AzuriteServer::fileURI() const {
+  return fmt::format(
+      "abfs://{}@{}.dfs.core.windows.net/{}", container_, account_, file_);
+}
+
+std::shared_ptr<const config::ConfigBase> AzuriteServer::hiveConfig(
+    const std::unordered_map<std::string, std::string> configOverride) const {
+  auto endpoint = fmt::format("http://127.0.0.1:{}/{}", port_, account_);
+  std::unordered_map<std::string, std::string> config(
+      {{"fs.azure.account.key.test.dfs.core.windows.net", key_},
+       {kAzureBlobEndpoint, endpoint}});
+
+  // Update the default config map with the supplied configOverride map
```

Comment:
Nit. Missing `.`. Or do we even need this comment?

Reply:
I updated the function documentation instead.